test.rs

```diff
@@ -0,0 +1,32 @@
+type Array<T, const N: usize> = [T; N];
```

Comment:
This should go in src/test/rustdoc somewhere (maybe `trait-visibility.rs` or something).